.env.example

```diff
@@ -1,9 +1,11 @@
 # General Settings
```

Comment:
Should keep this as the main "template file" and remove the other template file.